activejob/lib/active_job/log_subscriber.rb

```diff
@@ -111,7 +137,11 @@ def discard(event)
 
     private
       def queue_name(event)
-        event.payload[:adapter].class.name.demodulize.remove("Adapter") + "(#{event.payload[:job].queue_name})"
+        adapter_name(event.payload[:adapter]) + "(#{event.payload[:job].queue_name})"
+      end
+
+      def adapter_name(adapter)
+        adapter.class.name.demodulize.remove("Adapter")
```

Comment:
```suggestion
        adapter.class.name.demodulize.delete_suffix("Adapter")
```

Sorry, just noticed this now.

Reply:
Fixed.